src/sass/_card.scss

```diff
@@ -13,9 +13,11 @@
     -webkit-background-size: cover;
     -moz-background-size: cover;
     -o-background-size: cover;
-    background-size: cover;
-    background-position: center center;
+    background-size: contain;
+    background-position: 50% center center;
     height: 10em;
+    width: 80%;
```

Comment:
we can get rid of these changes since they no longer apply

Reply:
Yup I'll go ahead and remove this now too.

Reply:
<img width="1125" alt="screen shot 2018-11-15 at 10 20 42 pm" src="https://user-images.githubusercontent.com/8063614/48595931-b8017d80-e924-11e8-992e-ff3f381b760c.png">

When removing the width it shifts the images a bit to the left outside of the margins.

Reply:
Yup that worked, just did another pull request with the reflected updates.